docs/conf.py

```diff
@@ -74,7 +74,12 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = []
+html_static_path = ["_static"]
+html_context = {
+    'css_files': [
+        '_static/theme_overrides.css',  # override wide tables with word wrap
+        ],
+     }
```

Comment:
This will cause tabs to keep working:

```suggestion
html_css_files = [
    'theme_overrides.css',  # override wide tables with word wrap
]
```